src/sparseml/core/recipe/stage.py

```diff
@@ -57,6 +85,11 @@ def evaluate(self, parent_args: RecipeArgs = None, shift: int = None):
     def create_modifier(
         self, framework: Framework, parent_args: RecipeArgs = None
     ) -> StageModifiers:
+        """
+        :param framework: the framework to create the modifiers for
+        :param parent_args: Optional RecipeArgs to use for evaluation
+        :return: the StageModifiers for the stage
+        """
```

Comment:
Important to clarify that this evaluates the recipe and instantiates the actual modifiers, A diagram of that would be helpful here or further up. This is an important function because we go from the RecipeModifiers which are just parameters to the actual Modifiers

Reply:
Is this better?